segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/ServiceStarter.java

```diff
@@ -62,6 +63,9 @@
 public final class ServiceStarter {
     //region Members
 
+    @VisibleForTesting
+    protected HealthServiceManager healthServiceManager;
```

Comment:
It's not a good practice to make your fields non-private (protected implies non-private). Some external actor can just modify this. 

I suggest you change it back to private and add `@Getter` to it - you can then expose it via the getter without risking that someone can modify it (i.e., replace it with some other instance).